dbt/parser.py

```diff
@@ -218,7 +218,7 @@ def parse_node(node, node_path, root_project_config, package_project_config,
     config_dict.update(config.config)
     node['config'] = config_dict
 
-    context = dbt.context.parser.generate(node, package_project_config,
+    context = dbt.context.parser.generate(node, root_project_config,
```

Comment:
ok, so in `dbt.context.common.generate`, we only want the root project config. can you also double check:

dbt/context/runtime.py line 94
dbt/node_runners.py line 377

if you do that then this is approved